game/src/game/components/Pickups/Pickup.cpp

```diff
@@ -24,6 +33,9 @@ void Pickup::OnTriggerEnter(const OnTriggerEvent& data)
 
 void Pickup::OnUpdate(const Timestep& delta_time)
 {
+    // rotate the powerup around its y axis.
+    // TODO: Ask Radu to help with rotating the powerups every frame.
+    transform_->RotateEulerDegrees(glm::vec3(20.f, 20.f, 0.f));
```

Comment:
For this, I think you could do something like:
```c++

// Rotate by 90 degrees/sec
static constexpr float kRotationSpeed = 90.0f;
transform_->RotateEulerDegrees(vec3(kRotationSpeed * delta_time.GetSeconds(), 0.0f, 0.0f)));
```

(Assuming I didn't do anything wrong with my Euler -> Quat conversion in that function, but I might've lol)